apps/generator/docs/deprecate-cli.js-ag.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Deprecate ag"
```

Comment:
this is migration document, so better call it like @Gmin2 his doc for migrating Nunjucks, so title related to content

also update name of file, and better not use `cli.js` in title, I'm not sure how extra `.` in the filename will be reflected in url of the document link in the website